op-deployer/pkg/deployer/opcm/read_superchain_deployment.go

```diff
@@ -0,0 +1,30 @@
+package opcm
+
+import (
+	"github.com/ethereum-optimism/optimism/op-chain-ops/script"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type ReadSuperchainDeploymentInput struct {
+	OPCMAddress common.Address `abi:"opcmAddress"`
```

Comment:
Could you use the standard name for [OpcmImpl](https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/addresses/contracts.go#L30) instead of OPCMAddress here?